pkg/handlers/upgrade_policies.go

```diff
@@ -0,0 +1,271 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/ocm-agent/pkg/consts"
+	log "github.com/sirupsen/logrus"
+)
+
+type UpgradePoliciesHandler struct {
+	ocm       *sdk.Connection
+	clusterId string
+}
+
+// https://api.openshift.com/#/default/get_api_clusters_mgmt_v1_clusters__cluster_id__upgrade_policies
+func NewUpgradePoliciesHandler(o *sdk.Connection, clusterId string) *UpgradePoliciesHandler {
+	log.Debug("Creating new upgrade policies Handler")
+	return &UpgradePoliciesHandler{
+		ocm:       o,
+		clusterId: clusterId,
+	}
+}
+
+func (g *UpgradePoliciesHandler) AddUpgradePolicy(clusterID string, policy *cmv1.UpgradePolicy) (*cmv1.UpgradePolicy, error) {
```

Comment:
Not a blocker, but good to have doc comments for public methods 
ref : https://tip.golang.org/doc/comment